wandb/sdk/internal/handler.py

```diff
@@ -788,7 +788,7 @@ def _handle_glob_metric(self, record: Record) -> None:
             self._metric_globs[metric.glob_name].CopyFrom(metric)
         else:
             self._metric_globs[metric.glob_name].MergeFrom(metric)
-        self._dispatch_record(record)
+        # self._dispatch_record(record)
```

Comment:
?
are globs handle totally in the handler now? does the transaction log have what it needs to do the same thing when in offline mode.

Reply:
let's do up to the sender since eventually we would want the backend to handle it...